samples/Samples/Areas/Identity/Pages/Account/ConfirmAccount.cshtml.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Amazon.AspNetCore.Identity.Cognito;
+using Amazon.Extensions.CognitoAuthentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Samples.Areas.Identity.Pages.Account
+{
+    [AllowAnonymous]
+    public class ConfirmAccountModel : PageModel
+    {
+        private readonly CognitoUserManager<CognitoUser> _userManager;
+
+        public ConfirmAccountModel(UserManager<CognitoUser> userManager)
+        {
+            _userManager = userManager as CognitoUserManager<CognitoUser>;
+        }
+
+        [BindProperty]
+        public InputModel Input { get; set; }
+
+        public string ReturnUrl { get; set; }
+
+        public class InputModel
```

Comment:
This is a small model, yes, but it is an independent part of MVC. It should be given it's own file.

Reply:
https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-2.2&tabs=visual-studio